circle.yml

```diff
@@ -1,20 +1,116 @@
-dependencies:
-  override:
-    - curl -L https://atom.io/download/deb -o atom-amd64.deb
-    - sudo dpkg --install atom-amd64.deb || true
-    - sudo apt-get update
-    - sudo apt-get -f install
-    - node --version
-    - npm --version
-    - atom --version
-    - rm -rf node_modules; npm install
-    - npm run compile
+version: 2
 
-test:
-  override:
-    - npm run lint
-    - npm test
+defaults: &defaults
+  working_directory: /tmp/project
+  docker:
+    - image: arcanemagus/atom-docker-ci:stable
+  steps:
+    # Restore project state
+    - attach_workspace:
+        at: /tmp
+    - run:
+        name: Update distro
+        command: |
+          sudo apt-get update && \
+          sudo apt-get -f install --assume-yes --quiet --no-install-suggests \
+            --no-install-recommends
+    - run:
+        name: Create VFB for Atom to run in
+        command: /usr/local/bin/xvfb_start
+    - run:
+        name: Atom version
+        command: ${ATOM_SCRIPT_PATH} --version
+    - run:
```

Comment:
This is unlikely to cause an issue here, but this should be moved after the `install` step.

Reply:
After install package dependencies?

Reply:
Yep, just like in the other CircleCI config updates 😉.

Reply:
hmmm, it is just like in the other CircleCI config updates, :confused:

Reply:
Ah whoops, I commented that [here](https://github.com/AtomLinter/linter-gfortran/pull/23#discussion_r254846381) before, but missed that it wasn't actually done before.

It's _very_ unlikely to cause issues, but we might as well get it right the first time here 😉.

Reply:
Aha! remember now!